### PR TITLE
PR for #104: enable options cluster and robust

### LINKED ISSUE #104: Check vce vs. cluster for IV estimation without reghdfe.

### PR DESCRIPTION
**Pull request by Constantino-Carreto-Romero**
Closes #104

### CODE REVIEW

**Comment by Constantino-Carreto-Romero** [comment]
Per call: @Constantino-Carreto-Romero will:
-  move the solution code into a program apart but inside `_eventiv`
- make more tests to check for cases when the code could break

**Comment by Constantino-Carreto-Romero** [comment]
In https://github.com/JMSLab/xtevent/pull/107/commits/4f1692461b28cf9866310a025dc9076a9450438f I moved part of the solution code to a parser at the end of the program. Thanks to the parser I'm not using regex functions anymore. I ran the test file as well.

File: xtevent/_eventiv.ado [hunk 1]
```diff
@@ -260,7 +260,41 @@ program define _eventiv, rclass
 				loc cmd "xtivreg"
 				loc ffe "fe"
 			}
-			`cmd' `varlist' (`proxy' = `leadivs' `varivs') `included' `tte' [`weight'`exp'] if `touse' , `ffe' `small' `options'
+			*translate standar error specification:
+			*analyze inclusion of cluster or robust in options
+			parse_es ,`options'
+			loc cl_orig = r(cl_orig)
```
Comment (jorpppp) [inline xtevent/_eventiv.ado:266]: @Constantino-Carreto-Romero we can loop over these options.

**Comment by Constantino-Carreto-Romero** [comment]
@jorpppp  in https://github.com/JMSLab/xtevent/pull/107/commits/b9cacdf08c8726563560493176db41153f47338a I added the loop for https://github.com/JMSLab/xtevent/pull/107#discussion_r981754896

I checked that everything keeps working after this little change.

**Comment by jorpppp** [comment]
Thanks @Constantino-Carreto-Romero. Merging now.

**Pull request merged**